apps-rendering/src/components/metadata.tsx

```diff
@@ -1,13 +1,15 @@
 // ----- Imports ----- //
 
 import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
 import {
 	Lines,
 	ToggleSwitch,
 } from '@guardian/source-react-components-development-kitchen';
 import { neutral, remSpace } from '@guardian/src-foundations';
 import { from, until } from '@guardian/src-foundations/mq';
 import { Design, Display } from '@guardian/types';
+import type { Format } from '@guardian/types';
```

Comment:
Given Olly's PR, this would now probably need to come from `guardian/libs`.

Reply:
Yeaa I just did a merge from main and got all Olly's changes